odoo/cli/server.py

```diff
@@ -136,6 +138,11 @@ def main(args):
         for db_name in preload:
             try:
                 odoo.service.db._create_empty_database(db_name)
+            except ProgrammingError as err:
+                if err.pgcode == errorcodes.INSUFFICIENT_PRIVILEGE:
+                    _logger.warning("Could not determine if database exists: %s" % err)
```

Comment:
I would rather (let the logger do the formatting) :
```python
_logger.warning("Could not determine if database %s exists, skipping auto-creation: %s", db_name, err)
```
or `_logger.info()`, as discussed.